### PR TITLE
05-descendant-combinator: Add full solution

### LINKED ISSUE #153: CSS Foundations Exercise 05-descendant-combinator: Add full solution
Complete the following REQUIRED checkboxes:
-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this issue follows the `location for request: brief description of request` format, e.g. `Foundations exercises: Add exercise for XYZ`

The following checkbox is OPTIONAL:

-   [x] I would like to be assigned this issue to work on it

<hr>

**1. Description of the Feature Request:**

Add full descendant combinator solution for this exercise to let us see the possibilities which we may have never thought of.

File Path: css-exercises>foundations>05-descendant-combinator>solution>solutions.css

**2. Acceptance Criteria:**

- [ ] Add to solutions.css file either by css comment or by code.

```
/* Other possible combinations of selectors

div p
div .text
.container p

*/
```
or 

```
div p {
  background-color: yellow;
  color: red;
  font-size: 20px;
  text-align: center;
}

div .text {
  background-color: yellow;
  color: red;
  font-size: 20px;
  text-align: center;
}

.container p {
  background-color: yellow;
  color: red;
  font-size: 20px;
  text-align: center;
}
```

### PR DESCRIPTION
**Pull request by Micosaur**
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `01-flex-center: Update self check`

<hr>

**1. Because:**

Closes #153 

**2. This PR:**

-   [ ] Add other possible solutions comment to solutions.css file

### CODE REVIEW

File: foundations/05-descendant-combinator/solution/solution.css [hunk 1]
```diff
@@ -3,4 +3,12 @@
...
\ No newline at end of file
+}
+
+/* Other possible combinations of selectors
+
+div p
+div .text
+.container p
+
+*/
```
Comment (thatblindgeye) [inline foundations/05-descendant-combinator/solution/solution.css:14]: Couple of things:

- Suggestion: changing line 8 to "Below are some other possible descendant combinators:"
- The file just needs an empty new line at the end of it

**Review by thatblindgeye — CHANGES_REQUESTED** [review]

File: foundations/05-descendant-combinator/solution/solution.css (hunk 1 above)
Comment (Micosaur) [reply foundations/05-descendant-combinator/solution/solution.css]: My mind was stuck in the "imperative mood" due to git commits. Haha

**Comment by Micosaur** [comment]
Added an empty new line at the end, but it doesn't show on github (Line 15). Please let me know if you see the changes on your end.

**Review by thatblindgeye — APPROVED** [review]

**Comment by thatblindgeye** [comment]
Ah that's fine. I don't see the symbol warning about the lack of a new line in your latest changes so it's most likely good. Thank you for making this update!

**Pull request merged**